packages/core/src/alert/alert-group.stories.ts

```diff
@@ -4,14 +4,18 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { getElementStorybookArgTypes, spreadProps, getElementStorybookArgs } from '@clr/core/internal';
+import { getElementStorybookArgTypes, spreadProps, getElementStorybookArgs, I18nService } from '@clr/core/internal';
 import { action } from '@storybook/addon-actions';
 import { html } from 'lit-html';
 import { ClarityIcons, headphonesIcon, nodeGroupIcon, timesCircleIcon } from '@clr/core/icon';
 import customElements from '../../dist/core/custom-elements.json';
 
 ClarityIcons.addIcons(headphonesIcon, nodeGroupIcon, timesCircleIcon);
 
+I18nService.localize({
+  alert: { closeButtonAriaLabel: 'close this alert' },
+});
```

Comment:
Small nit, we should probably import the `I18nService` from `@clr/core` here just for helping set an example for anyone looking at the example code in storybook